app/forms/update_user_password_form.rb

```diff
@@ -48,6 +49,7 @@ def extra_analytics_attributes
       active_profile_present: user.active_profile.present?,
       pending_profile_present: user.pending_profile.present?,
       user_id: user.uuid,
+      required_password_change: required_password_change.present?,
```

Comment:
I don't think it's necessary to `.present?` here since we always provide a default value. Might also think it'd be a good idea to add the same `?` predicate method suffix method alias here as well.

```suggestion
      required_password_change: required_password_change?,
```